app/assets/stylesheets/components/_url-preview.scss

```diff
@@ -2,6 +2,10 @@
   @include govuk-font(19);
 
   @include govuk-responsive-margin(6, "bottom");
+
+  .govuk-inset-text {
+    margin-top: 0;
```

Comment:
This one confused me a bit - as discussed, it only applies in narrow width views.

Reply:
After the url preview component markup on the edit page has been move on a new row the margin-top of the inset text element creates unnecessary whitespace between it and the title field above (left). We do not want this as the page address should be visually associated with the title so we're reducing the spacing to match the original (right).

I updated the commit description too trying to explain the need for it.

![url-preview-margin](https://user-images.githubusercontent.com/788096/61368486-d938b600-a885-11e9-9980-4d5d164520ea.jpg)